synapse/cortex.py

```diff
@@ -5830,14 +5830,19 @@ async def count(self, text, opts=None):
         opts = self._initStormOpts(opts)
 
         if self.stormpool is not None and opts.get('mirror', True):
-            extra = await self.getLogExtra(text=text)
             proxy = await self._getMirrorProxy(opts)
 
             if proxy is not None:
-                logger.info(f'Offloading Storm query {{{text}}} to mirror.', extra=extra)
+                proxname = proxy._ahainfo.get('name')
+                extra = await self.getLogExtra(mirror=proxname, hash=s_storm.queryhash(text))
+                logger.info(f'Offloading Storm query to mirror {proxname}.', extra=extra)
 
                 mirropts = await self._getMirrorOpts(opts)
 
+                mirropts.setdefault('_loginfo', {})
+                mirropts['_loginfo']['pool:to'] = proxname
```

Comment:
Does the `pool:to` even make sense since this is going to be logged on the "proxname" host?

Reply:
I think we can probably omit it as redundant :+1: